docs/iris/example_code/General/cube_from_scratch.py

```diff
@@ -0,0 +1,83 @@
+"""
+Building Iris Cubes From Scratch
+================================
+
+This example shows a method to make an Iris cube from nothing. 
+A synthetic data-set is created for the cube and so are a set of latitude, longitude and time coordinates. 
+All of these are folded into the cube. 
+Lastly, the code plots an output image to represent the synthetic cube.
+
+"""
+
+
+import numpy as np
+import iris
+import iris.quickplot as qplt
+import matplotlib.pyplot as plt
+import cf_units
+
+
+def main():
+    # Here we create our lists for latitude and longitude.
+    latitudes = [0, 10, 20, 30, 40]
+    longitudes = [5, 15, 25, 35, 45]
+
+    # Now we generate an array of random values. These values will be the basis for our cube's data.
+    data = (np.random.rand(3, 5, 5))
+
+    print data
```

Comment:
In order to run in Python3 as well as Python2 this would have to be
```
print(data)
```

However, I would prefer not to have a print line in an example, so I suggest simply removing the line.